src/cdomains/mHP.ml

```diff
@@ -24,7 +24,6 @@ let current (ask:Queries.ask) =
   }
 
 let pretty () {tid; created; must_joined} =
-  let tid_doc = Some (Pretty.dprintf "tid=%a" ThreadIdDomain.ThreadLifted.pretty tid) in
```

Comment:
Whys is the tid of the thread not interesting here?

Reply:
See the example in #1192. MHP takes `tid` directly by querying the thread ID analysis, but that analysis already shows this exact information. So thread IDs are always duplicated in race output.